exp-player/addon/components/exp-lookit-story-page/component.js

```diff
@@ -336,6 +347,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         }
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
Yes, that's a better plan longer-term. It might really be as simple as giving control over font size, but right before heading out for a while, I figured it might be better to cover all possible bases. It's on my list now!